code/datums/weather/weather_types.dm

```diff
@@ -184,3 +184,33 @@
 		status_signal.data["picture_state"] = "radiation"
 
 	frequency.post_signal(src, status_signal)
+
+
+/datum/weather/acid_rain
+	name = "acid rain"
+	desc = "Some stay dry and others feel the pain"
+
+	telegraph_duration = 400
+	telegraph_message = "<span class='danger'>Stinging droplets start to fall upon you..</span>"
+	telegraph_sound = 'sound/lavaland/ash_storm_windup.ogg'
```

Comment:
It's supposed to be ambient, I think. Could be just rain sounds and a sizzling sound effect to play for when people/stuff is being subject to the acid, besides nature which can resist it.